src/gatsby/src/templates/site.js

```diff
@@ -32,6 +33,22 @@ import Seo from "../components/seo"
 
 import { StyledGrid } from "../utils/constants"
 
+const qs = require('qs')
+
+const loadProtectedArea = (apiBaseUrl, orcsId) => {
```

Comment:
You can load protected areas by orcs using the findOne() operation onstead of findMany() like this:

https://cms.bcparks.ca/api/protected-areas/8

The default behaviour was overwritten in the controller code by the previous dev team.